__tests__/app.test.js

```diff
@@ -149,6 +149,27 @@ describe('"GET /api/articles', () => {
       });
   });
 
+  test("200: Accepts a topic query which filters articles by the topic specified in the query", () => {
+    return request(app)
+      .get("/api/articles?topic=mitch")
+      .expect(200)
+      .then(({ body: { articles } }) => {
+        expect(articles.length).toBe(12);
+        articles.forEach(({topic}) => {
+          expect(topic).toBe("mitch")
+        })
+      });
+  });
+
+  test("404: Sends an appropriate status and error message when given a non-existent topic", () => {
```

Comment:
good! What about when a topic exists but doesn't return any articles? For example, paper?